docs/UsingSpiffdemo/Basic_Tools.md

```diff
@@ -215,14 +208,66 @@ during the process execution.
 
 
 ## Writing Scripts 
+Writing scripts refers to the process of creating custom code or scripts to increase the functionality and automation of a software application or system. 
+
+In SpiffArena, writing scripts involves creating code snippets or 
+specific tasks, manipulate data, or handle complex logic.
+
+The scripting language used for writing scripts in SpiffArena is  Python, a widely used programming language. Python provides a rich set of libraries, frameworks, and tools that facilitate script development, making it a popular choice for implementing custom logic and automation.
+
+Let's explore an example of a Script Task in our basics section:
+
+1. **Start Event and User Task - "Form"**
+
+The process begins with a Start Event, followed by a User Task named "Form." Users will fill out the form, and the data will be passed to the next task, which is a Script Task.
+![](Images/Script_Example_S2.png)
+
+2. **Script Task to collect data**
+
+In the Script Task, we have created a script that collects three variables from the form and calculates a score based on certain conditions. The score is then stored in the "score" variable.
+
+![](Images/Script_Example_S3.png)
 
+Here's the script:
+
+![](Images/Script_Example_S4.png)
```

Comment:
This script is not legible. Consider presenting the code in a code block rather than as a picture.  See https://mystmd.org/guide/code